### ISSUE TITLE
IMPORTERROR: No module named 'utils'

### ISSUE THREAD

**Issue by aparnara** [laplacesdemon/django-social-friends-finder#8]
Python 3.3.2, Django 1.5.1 - running synDB after adding social_friends_finder to the installed apps list returns IMPORTERROR: No module named 'utils'. 

BTW I am trying to get this working with django-allauth which is installed and working. 

Can someone tell me to do? Thanks!

**Comment by procmail** [comment]
Yeah same error. I see this ticket has been opened from July til now, and no response.

**Comment by oxbits** [comment]
I am getting the same error in python-social-auth
